_pages/debrief.md

```diff
@@ -78,6 +80,7 @@ After all interviews are complete, open the floor for any additional discussion:
 
 > Now that you’ve heard everyone’s interview debrief, would anybody like to change their scores, or provide additional comments and thoughts?
 
+As a reminder, the scoring numbers are used to determine relative confidence and are primarily tools to engage interviewers in a slow, deliberate thought process. Keep track of their score changes, and ask why they have changed — it is useful information (and is likely just as important if not more important than the score itself).
```

Comment:
```suggestion
As a reminder, the scoring numbers are used to determine relative confidence and are primarily tools to engage interviewers in a slow, deliberate thought process. Keep track of their score changes, and ask why they have changed — it is useful information (and is likely just as important if not more important than the score itself). It may be helpful to remind candidates that re-scoring should be scoped to their own interview topics, (e.g., "I heard evidence from other interviews that makes me reinterpret the data I used to score my interview"), and not an overall impression of the candidate from data gathered from other interviews.
```